modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/Parameter.java

```diff
@@ -16,93 +16,113 @@
 
 package io.swagger.oas.annotations;
 
-import io.swagger.oas.annotations.enums.Explode;
-import io.swagger.oas.annotations.media.ArraySchema;
-import io.swagger.oas.annotations.media.Content;
-import io.swagger.oas.annotations.media.Schema;
-import io.swagger.oas.annotations.parameters.Parameters;
-
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Inherited;
-import java.lang.annotation.Repeatable;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+import io.swagger.oas.annotations.enums.Explode;
+import io.swagger.oas.annotations.media.ArraySchema;
+import io.swagger.oas.annotations.media.Content;
+import io.swagger.oas.annotations.media.Schema;
+
 /**
- * 
- *
- * 
+ * Describes a single operation parameter
  **/
-
-
 @Target({ ElementType.PARAMETER,
           ElementType.METHOD })
-@Repeatable(Parameters.class)
 @Retention(RetentionPolicy.RUNTIME)
 @Inherited
 public @interface Parameter {
   /**
-   * the name of the parameter
+   * The name of the parameter.
+   * @return the parameter's name
    **/
   String name() default "";
 
   /**
-   * the location of the parameter.  Ignored when empty string
+   * The location of the parameter.  Possible values are "query", "header", "path" or "cookie".  Ignored when empty string.
+   * @return the parameter's location
    **/
   String in() default "";
```

Comment:
I agree @webron  I opened this new issue for this: https://github.com/swagger-api/swagger-core/issues/2371